python/hyperon/atoms.py

```diff
@@ -447,7 +617,7 @@ def merge_into(self, input: Union['BindingsSet', Bindings]):
             hp.bindings_set_merge_into(self.c_set, new_set.c_set);
 
     def iterator(self):
-        """Gets Bindings set iterator"""
+        """Iterates through all Bindings frames."""
```

Comment:
As well, this function returns an iterator (indicating this would be more precise than saying that it iterates).